app/products/calls/connection/connection.ts

```diff
@@ -46,9 +47,11 @@ export async function newConnection(serverUrl: string, channelID: string, closeC
         logError('Unable to get media device:', err);
     }
 
+    InCallManager.start({media: 'audio'});
+    InCallManager.stopProximitySensor();
```

Comment:
Curious to know what's the context on this move.

Reply:
I was experimenting with where InCallManager initializes the audio system. But I think your implication is right, better to leave that for the audio rework PR (when I figure it out).